sagemaker-clarify/bias-and-explainability/student_performance_clarify.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    HTML('Click here to view datasets in S3'.format(url))
</pre><p>Can you print the S3 path too, in case a user wants to retrieve it via the CLI or boto3 rather than a console link? (Applies to all instances where you provide a console link to outputs.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>